### ISSUE TITLE
Could you provide a download link for the pre-training dataset?

### ISSUE THREAD

**Issue by huha6** [VarunUllanat/mint#24]
Hello!  Great work! 
We are trying to reproduce your work, but we are a bit concerned that the data we processed for pre-training the backbone might not match your original training data.  Therefore, we would like to ask if you could provide a download link for the pre-training dataset to ensure data consistency.

**Comment by huha6** [comment]
@VarunUllanat

**Comment by VarunUllanat** [comment]
Hello, did you follow the steps in the README [here](https://github.com/VarunUllanat/mint?tab=readme-ov-file#pre-training-on-string-db)?

**Comment by ddofer** [comment]
Hi, along those lines: You wrote that you filtered your pretraining data to avoid any overlap/leakage with the Gold/Bennet PPI evaluation data. Where is that done in your code/the instructions? I couldn't find it! 
Thanks

**Comment by VarunUllanat** [comment]
Hello! We didn’t filter the pretraining data against the Gold/Bernett PPI dataset. Pretraining is fully unsupervised (MLM over protein pairs) and doesn’t use the Bernett labels or negatives. 

The overlap/leakage refers to how the supervised training and test sets were created in the original [Bernett et al. paper](https://academic.oup.com/bib/article/25/2/bbae076/7621029).

**Comment by ddofer** [comment]
Got it! Thanks

**Comment by hypefolder** [comment]
Hi @VarunUllanat, thanks for open-sourcing your work!

I this not a possible source of data leakage in evaluation? I am a bit confused as to how the evaluation is fair if you pre-train on basically all positive PPI data and then test sets of sequences that likely overlap with your pre-training data.
